src/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(images/star.png);
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(images/star-active.png);
+}
+
+.stars--2 .stars__star:nth-child(-n +  2) {
+  background-image: url(images/star-active.png);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(images/star-active.png);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.png);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.png);
+}
```

Comment:
Try to rewrite this
<img width="1122" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/389bbefe-a328-4d02-ba57-7309d190a7d4">